src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/Sdk/AutoImport.props

```diff
@@ -35,8 +35,14 @@ https://github.com/dotnet/designs/blob/4703666296f5e59964961464c25807c727282cae/
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\font\*.otf" />
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\font\*.ttc" />
     <AndroidResource Include="$(MonoAndroidResourcePrefix)\raw\*" Exclude="$(MonoAndroidResourcePrefix)\raw\.*" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(MonoAndroidAssetsPrefix)' != '' and '$(EnableDefaultAndroidItems)' == 'true' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '8.0')) ">
     <!-- Default Asset file inclusion -->
     <AndroidAsset Include="$(MonoAndroidAssetsPrefix)\**\*" Exclude="$(MonoAndroidAssetsPrefix)\**\.*\**" />
```

Comment:
[Similar to above](https://github.com/xamarin/xamarin-android/pull/7837/files#r1120937802), but…

Why support `$(MonoAndroidAssetsPrefix)` at all anymore?  Consider `@(TransformFile)`,  `@(AndroidLibrary)`, `@(AndroidJavaSource)`, etc., below: *none* of those use `$(MonoAndroidAssetsPrefix)`, and thus aren't susceptible to pulling in `C:\**\*`.

*Alternatively*, perhaps we could *require*/force `$(MonoAndroidAssetsPrefix)` to *end with* "directory-separator-char", which would allow us to change these includes to e.g.:

```xml
<AndroidResource Include="$(MonoAndroidAssetsPrefix)**\*" />
```

which would *also* prevent us from trying to traverse `C:\**\*`.

Reply:
After discussion and thinking about `AutoImports.props` MSBuild ordering, we decided to just format my code better...

Spec if needed: https://github.com/dotnet/designs/blob/main/accepted/2020/workloads/workload-resolvers.md#workload-props-files